Update knative function controller (#5121)
kyma-project/kyma: components/knative-function-controller/README.md

### Run locally
Follow these steps to run the Knative Function controller locally:

> 1. Create a ServiceAccount to enable knativeBuild docker builds

Reviewer: ```suggestion
1. Create a ServiceAccount to enable knativeBuild Docker builds.
```

> Currently you cannot run builds in every existing namespace of your cluster. ServiceAccounts with linked docker 

Reviewer: ```suggestion
Currently, you cannot run builds for every Namespace existing in your cluster. 
```

Author: the build cannot run if it is created IN a certain namespace, as it requires a serviceaccount in that namespace

Reviewer: Maybe we can combine these two and have something like this then:

`To run a build in a Namespace, create ServiceAccounts with linked Docker repository credentials for each Namespace that will be used for the knative-function controller.`

> repository credentials have to be created for each namespace that will be used for the knative-function controller.

Reviewer: ```suggestion
Create ServiceAccounts with linked Docker repository credentials for each Namespace that will be used for the knative-function controller.
```
